pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -17975,5 +17975,17 @@ final: prev:
     meta.homepage = "https://github.com/jhradilek/vim-snippets/";
   };
 
+  tailwind-tools-nvim = buildVimPlugin {
+    pname = "tailwind-tools-nvim";
```

Comment:
```suggestion
    pname = "tailwind-tools.nvim";
```
This should be the original name of the plugin.

Reply:
OK fixing

Reply:
Fixed